src/pages/signIn/SignIn.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Formik } from 'formik';
+
+import { RegularText, SignForm, SignInButton, StyledLink, StyledTitle, SignInForm } from './styled';
+
+import { getUserInfo } from '../../redux/actions/actionCreators';
+import { initValuesForLogIn, ValidSchemeForLogIn } from './validation';
+import { useQueryToLogin } from '../../hooks/query/loginQuery';
+import { fetchUsers } from '../../services/userService';
+import ValidInput from './validInput';
+
+const SignIn = () => {
+    const [userCredentials, setUserCredentials] = useState({});
+    const dispatch = useDispatch();
+
+    const { data } = useQueryToLogin('currentUser', () => fetchUsers(), {
```

Comment:
Are you kidding me? I spent 30 minutes with you yesterday to create the useUsers react-query, but you just removed almost all the stuff............